polyfills/Array/prototype/includes/tests.js

```diff
@@ -1,13 +1,26 @@
 /* eslint-env mocha, browser */
 /* global proclaim */
 
+var supportsDescriptors = Object.defineProperty && (function () {
+	try {
+			var obj = {};
```

Comment:
These are tabbed in a bit too much. Shouldn't stop merging but it doesn't match the style of the rest of the code